fastlane/lib/fastlane/actions/git_default_remote_branch.rb

```diff
@@ -0,0 +1,57 @@
+module Fastlane
+  module Actions
+    class GitDefaultRemoteBranchAction < Action
```

Comment:
I think, we can `omit` Default from this action name now...because now this action accepts custom `remote_name` input param, thoughts? 🤔

Reply:
Works for me & already implemented in most recent commit 👍